Jenkinsfile

```diff
@@ -115,73 +108,47 @@ pipeline {
             }
         }
 
-        stage('Generate front ressources') {
+        stage('Deploy to Nexus') {
```

Comment:
Je crois que c'est ce qu'on appelle "Publish" dans la terminologie actuelle. On devrait peut-être garder le terme ? Cela risque d'être confondu avec le vrai déploiement vers les environnements bac à sable.

Reply:
C'est pour reprendre le terme maven. Et je précise bien "to Nexus", ce qui devrait éviter de confondre avec un déploiement sur un env.

Reply:
Ok, c'est juste un terme de toute façon.